app/controllers/api/v1/admin/roles_controller.rb

```diff
@@ -63,6 +63,9 @@ def update
         # Returns: { data: Array[serializable objects] , errors: Array[String] }
         # Does: Deletes a role.
         def destroy
+          default_roles = @role.name == 'User' || @role.name == 'Administrator' || @role.name == 'Guest'
+          return render_error errors: @role.errors.to_a, status: :bad_request if default_roles
```

Comment:
I would return a `:forbidden` here instead since `@role.errors.to_a` wil be blank